Services/CO.CDP.Organisation.WebApi.Tests/UseCase/AssignIdentifierUseCaseTest.cs

```diff
@@ -211,6 +211,160 @@ public async Task ItAssignsNonPrimaryIdentifierIfTheOrganisationAlreadyHasIdenti
         result.Should().BeTrue();
     }
 
+    [Fact]
+    public async Task ItAssignsPPONPrimaryIdentifierIfOnlyInternationalIdentifierExists()
+    {
+        var organisation = GivenOrganisationExist(
+            organisationId: Guid.NewGuid(),
+            identifiers:
+            [
+                new Persistence.Organisation.Identifier
+                {
+                    Primary = true,
+                    Scheme = "FR-COH",
+                    IdentifierId = "944432342",
+                    LegalName = "France Acme Ltd"
+                }
+            ]);
+
+        var result = await UseCase.Execute(new AssignOrganisationIdentifier
+        {
+            OrganisationId = organisation.Guid,
+            Identifier = new OrganisationIdentifier
+            {
+                Id = "c0777aeb968b4113a27d94e55b10c1b4",
+                Scheme = "GB-PPON",
+                LegalName = "France Acme Ltd"
+            }
+        });
+
+        _organisations.Verify(r => r.Save(It.Is<Persistence.Organisation>(o =>
+            o.Guid == organisation.Guid && o.Identifiers.Contains(new Persistence.Organisation.Identifier
+            {
+                Primary = true,
+                Scheme = "GB-PPON",
+                IdentifierId = "c0777aeb968b4113a27d94e55b10c1b4",
+                LegalName = "France Acme Ltd"
+            }
+            ))));
```

Comment:
I'd include one more assertions in this and following test cases - that there's only one primary identifier.

Reply:
done!!